Dockerfile

```diff
@@ -1,49 +1,50 @@
-FROM	debian:wheezy
-
+FROM	   node:6.7.0-slim
```

Comment:
From what I gathered in discussion, they aren't adding any packages - merely setting up a local mirror for `apt` and `npm`

It would be nice to have a way to test our Dockerfile without actually deploying but I think it's probably okay anyway based on what they told me.

In a short bit we might be able to get rid of that `env-config.sh` and then that would remove all of these uncertain implicit assumptions.